frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -106,6 +105,7 @@ export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
                   disabled={editInfo.identification}
                   type="number"
                   field={withNameSpace('latitude')}
+                  required
```

Comment:
These were required in previous steps, so I'm just adding the "required" styling in the review form to match